overviews/core/_posts/2012-09-20-futures.md

```diff
@@ -615,8 +615,30 @@ Invoking the `future` construct uses a global execution context to start an asyn
 Support for extending the Futures API with additional utility methods is planned.
 This will allow external frameworks to provide more specialized utilities.
 
+
 ## Blocking
 
```

Comment:
Please add a short one-two lines introductory paragraph - something like the following:

    Futures are generally asynchronous and do not block the underlying execution threads.
    However, in certain cases, it is necessary to block.
    We distinguish two forms of blocking the execution thread:
    invoking arbitrary code that blocks the thread from within the future,
    and blocking from outside another future, waiting until that future gets completed..